tools/cldr-apps/src/main/webapp/css/redesign.css

```diff
@@ -1079,8 +1070,8 @@ html {
 
 #itemInfo div.xpath {
 	position: fixed;
-	/* TODO: get the right width of the "slider" for left sidebar; cf. beware-left-sidebar */
-	left: 34px;
+	/* The left sidebar "slider" width is 2% ( = 20% x 10%); cf. beware-left-sidebar */
+	left: 2.1%;
```

Comment:
I haven't done a ton with them, but i know sass/less precompilers lets you put math into CSS.

Reply:
less would be nice, also for declaring re-usable constants!

also, flex is good for avoiding the need for one element to know the size of another element -- the left sidebar would be better with flex, and it could push the main window to the right instead of overlapping it...